packages/frontend/src/hooks/tableVisualization/useTableConfig.ts

```diff
@@ -136,7 +144,8 @@ const useTableConfig = (
             canUseMetricsAsRows &&
             metricsAsRows &&
             resultsData?.metricQuery &&
-            pivotDimensions
+            pivotDimensions &&
```

Comment:
Was this throwing a type error without this? I was trying to get most logic under `canUseMetricsAsRows` unless there was a type error here

Reply:
removed, bad rebase